src/core_modules/capture-core/metaDataStoreLoaders/maintenance/usersCacheMaintenance.js

```diff
@@ -19,6 +18,7 @@ async function addUserCacheToHistory(
     mainStorageController: StorageController,
     userStorageController: StorageController,
 ) {
+    const { parentStoreNames: mainStores } = getContext();
```

Comment:
I agree that parentStoreNames is very vague, changed it to applicationStoreNames. I like to keep the other one as is, because that is the one you really should be using in this context (if not doing something out of the ordinary like here).

Reply:
After discussing it we said that the `applicationStorageNames` will be extracted from the context and then only the store names will stay.

We decided to follow up this on the last PR 

++

Reply:
also this looks like it is fixed here?